oak_functions/sdk/oak_functions/tests/integration_test.rs

```diff
@@ -141,9 +142,8 @@ async fn test_write_log() {
 
 #[tokio::test]
 async fn test_storage_get_item() {
-    let entries = hashmap! {
-       b"StorageGet".to_vec() => b"StorageGetResponse".to_vec(),
-    };
+    let entries =
+        HashMap::from_iter([(b"StorageGet".to_vec(), b"StorageGetResponse".to_vec())].into_iter());
```

Comment:
Not sure the from_iter / into_iter is actually doing anything here.

Reply:
This is the simplest way I found to convert an array of tupples into a hashmap. Even though I though it should work based on the documentation (https://docs.rs/hashbrown/latest/hashbrown/struct.HashMap.html#method.from) using `from` or `into` does not work. The compiler complains that "the trait bound `hashbrown::HashMap<Vec<u8>, Vec<u8>>: From<[(Vec<u8>, Vec<u8>); 1]>` is not satisfied".